feature/home/src/main/java/com/epmedu/animeal/home/HomeScreenUI.kt

```diff
@@ -101,6 +109,18 @@ private fun mapView(mapboxPublicKey: String, mapBoxStyleUri: String): MapView {
         MapView(context, mapInitOptions)
     }
 
+    // For testing purposes
+    AddMarker(
```

Comment:
I assume for tesing purpose we need a lot of markers with different combination status and animal type.
In scope of task we need to check if everything works as expected and we can handle markers on map without any problems.

I can propose following solution:
- create ViewModel with state of list markerts
- "load" markers from repository as stub data
- subscribe for state inside Composable screen

Reply:
I'm on it